lib/themes/dosomething/paraneue_dosomething/scss/content/_modal-crop.scss

```diff
@@ -7,22 +7,19 @@
   .image-editor {
     padding-top: 10px;
 
-    a {
-      color: $off-black;
-      text-decoration: underline;
-    }
-
     .__buttons {
       text-align: right;
 
-      .-rotate {
+      * {
```

Comment:
Oh I was just trying to get all the elements in this div to be inline-blocks so they display next to each other. But I can update to not use a global selector. Sorry!